docs/content/manual/pre-release/node/improve-node-failure-handling.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Improve Node Failure Handling By Automatically Force Delete Terminating Pods of StatefulSet/Deployment On Downed Node
+---
+1. Setup a cluster of 3 nodes
```

Comment:
`3 worker nodes` instead of `3 nodes`
The s.no of test steps should be correct.

Reply:
> 3 worker nodes instead of 3 nodes
Thank you. Fixed

> The s.no of test steps should be correct.
It will be correct when we see `view file` or when it displayed on the website